packages/central-server/src/apiV2/index.js

```diff
@@ -75,7 +75,12 @@ import {
   GETMapOverlayGroupRelations,
```

Comment:
Oh almost forgot, lets get rid of the old CRUD routes. EditSurveyResponses, EditAnswers, DeleteAnswers.

Reply:
Thanks!